family/app/controllers/people_controller.rb

```diff
@@ -93,4 +110,14 @@ def person_params
       ]
     )
   end
+
+  #selects all people that are not friends with @person
+  def select_possible_friends
+    @possible_friends = Person.all.to_a - @person.friends.to_a
```

Comment:
could you not make `@possible_friends = Person.all.to_a - @person.friends.to_a - [@person]`?